app/model/Model.scala

```diff
@@ -37,37 +37,38 @@ case class Edit(line1: String, line2: Option[String], line3: Option[String], tow
 
 object JourneyConfigDefaults {
 
-  val CONFIRM_PAGE_TITLE = "Confirm Address"
-  val CONFIRM_PAGE_HEADING = "Confirm Address"
+  val CONFIRM_PAGE_TITLE = "Company address"
```

Comment:
This seems like the wrong value for a default -- too SCRC specific

Reply:
Looking into it